src/libraries/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -70,6 +70,13 @@ public static void WindowWidth_GetUnix_Success()
         Helpers.RunInRedirectedOutput((data) => Console.WriteLine(Console.WindowWidth));
     }
 
+    [Fact]
+    [PlatformSpecific(TestPlatforms.iOS | TestPlatforms.MacCatalyst | TestPlatforms.tvOS)]  // Expected behavior specific to Unix
```

Comment:
Why is Browser supposed to not throw here but throw for WindowWidth?

Reply:
`[PlatformSpecific(TestPlatforms.AnyUnix & ~TestPlatforms.Browser)]` was original test condition.

Reply:
https://github.com/dotnet/runtime/commit/0de50030b876e9c77368e6144e53ed1b540a517c is the commit where original condition introduced.